### PR TITLE
file-roller: slight hardening

### PR DESCRIPTION
**Pull request by glitsj16**

### CODE REVIEW

File: etc/profile-a-l/file-roller.profile [hunk 1]
```diff
@@ -30,6 +31,7 @@ nogroups
 noinput
 nonewprivs
 noroot
+noprinters
```
Comment (kmk3) [inline etc/profile-a-l/file-roller.profile:34]:
```suggestion
noprinters
noroot
```

Sort.

**Review by kmk3 — COMMENTED** [review]
Could you try implementing the suggestion with `git commit --amend` as
mentioned here?

* <https://github.com/netblue30/firejail/pull/5711#issuecomment-1455061420>

`git commit --amend` just redoes the current commit with the changes
added/removed by `git add`.

For branches with a single commit, this should be simpler than making another
commit and squashing, for example.

File: etc/profile-a-l/file-roller.profile (hunk 1 above)
Comment (glitsj16) [reply etc/profile-a-l/file-roller.profile]: Fixed. Thanks!

File: etc/profile-a-l/file-roller.profile [hunk 2]
```diff
@@ -21,6 +21,7 @@ include whitelist-var-common.inc
 
 apparmor
 caps.drop all
+ipc-namespace
```
[the remark this answers is not in the document]
Comment (glitsj16) [reply etc/profile-a-l/file-roller.profile:24]: Not sure what you're referring to. Can you elaborate or provide an example of why adding this is opening up any holes in the sandbox? The profile also has private-bin, hence blocks any 'random' GUI application which isn't part of the allowed set of programs.

[the remark this answers is not in the document]
Comment (glitsj16) [reply etc/profile-a-l/file-roller.profile]: > I think that the point is the opposite; that adding this entry could break
integration with other tools.

Oh I see. Thank for clearing things up. I reverted this part of the PR.

**Review by kmk3 — APPROVED** [review]

**Pull request merged**